tests/phpunit/tests/db.php

```diff
@@ -86,7 +86,7 @@ public function test_locale_floats() {
 		// Switch to a locale using comma as a decimal point separator.
 		$flag = setlocale( LC_ALL, 'ru_RU.utf8', 'rus', 'fr_FR.utf8', 'fr_FR', 'de_DE.utf8', 'de_DE', 'es_ES.utf8', 'es_ES' );
 		if ( false === $flag ) {
-			$this->markTestSkipped( 'No European locales available for testing.' );
+			$this->fail( 'No European locales available for testing.' );
```

Comment:
Again: A contributor could be running the tests locally and not have the European locale available. Marking a test as skipped in such a case is the appropriate way to handle this. Not failing the test suite.